### PR TITLE
Fixing Dockerfile definition.

### PR DESCRIPTION
**Pull request by hbulgarini**
The docker image builder is failing with the following errors:

https://github.com/paritytech/parity-bridges-ui/runs/3485843336?check_suite_focus=true

In order to fix it i have change the order of the copy of the files and i added python package so errors are gone.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,12 +6,15 @@ WORKDIR /parity-bridges-ui
...
+ENV PYTHONUNBUFFERED=1
+RUN apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python
+RUN python3 -m ensurepip
+RUN pip3 install --no-cache --upgrade pip setuptools
 
-# install app dependencies
-COPY package.json .
-COPY yarn.lock .
-RUN yarn
 COPY . .
```
Comment (tomusdrw) [inline Dockerfile:15]: The `COPY` stuff should stay as-is, this is a common practice to utilize docker cache

File: Dockerfile [hunk 2]
```diff
@@ -6,12 +6,15 @@ WORKDIR /parity-bridges-ui
 
 RUN apk update
 RUN apk add git
+# Install python/pip
+ENV PYTHONUNBUFFERED=1
+RUN apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python
```
Comment (tomusdrw) [inline Dockerfile:11]:
```suggestion
# Install git & python/pip
RUN apk add git python3
RUN ln -sf python3 /usr/bin/python
ENV PYTHONUNBUFFERED=1
```

File: Dockerfile [hunk 3]
```diff
@@ -6,12 +6,15 @@ WORKDIR /parity-bridges-ui
...
+RUN python3 -m ensurepip
+RUN pip3 install --no-cache --upgrade pip setuptools
 
-# install app dependencies
-COPY package.json .
-COPY yarn.lock .
-RUN yarn
 COPY . .
+RUN yarn
+RUN yarn downloadCustomTypes
```
Comment (tomusdrw) [inline Dockerfile:17]: Isn't this part of pre-build or post-install?

Comment (hbulgarini) [reply Dockerfile:17]: pre build, it downloads the custom types files prior to build the image.

Comment (tomusdrw) [reply Dockerfile:17]: right, so this line seems unneeded then :)

File: Dockerfile (hunk 1 above)
Comment (hbulgarini) [reply Dockerfile:15]: the problem i'm facing is that i need the path `src/configs` for making the `src/configs/chainsSetup/downloadCustomTypes.js` available, so i need more than the package.json and yarn.lock.

What's the best way to achieve this? Just copy the `src/configs` ? (i'm not an expert on this honestly)

Comment (hbulgarini) [reply Dockerfile]: ok! reverted ... the prebuild script solved the issue i had.

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**